app/_hub/kong-inc/openid-connect/overview/_index.md

```diff
@@ -927,6 +927,22 @@ We can use the output in `Authorization` header.
    }
    ```
 2. Done.
+   
+#### Test the JWT Access Token Authentication with access token in query string 
+
+To specify the bearer token as a query string parameter 
+
+```bash
+http -f patch :8001/plugins/5f35b796-ced6-4c00-9b2a-90eef745f4f9 \
+  config.bearer_token_param_type=query                          \
+  config.auth_methods=bearer                                     \
+  config.auth_methods=password # only enabled for demoing purposes
```

Comment:
```suggestion
curl -i -X PATCH http://localhost:8001/plugins/5f35b796-ced6-4c00-9b2a-90eef745f4f9  \
  --data "config.bearer_token_param_type=query"                 \
  --data "config.auth_methods=bearer"                           \
  --data "config.auth_methods=password" # only enabled for demoing purposes
```
Converting to cURL; at the moment, the OIDC doc is the only one that still uses HTTPie, but the rest of our docs use cURL. Converting the whole doc to cURL is in progress.